### ISSUE TITLE
Script for inference

### ISSUE THREAD

**Issue by amahtani** [jingdao/IR_detection#7]
Hi, thanks for open-sourcing this repo.

Is there a script to use the code on unknown dataset with no labels to test the model ?
The code test.py needs the label images to test the model...

I tried modifying it (in a new inference.py script) but the prediction line does not return the same output as in test.py:
result = sess.run(
        [net.logits_tf],
        {net.input_pl:input_images})

In that case, what would be the changes in the predicted_softmax line :
predicted_softmax = scipy.special.softmax(result[0,:,:,:],axis=-1)[:,:,1]

Thanks for any answers that may help !

**Comment by jingdao** [comment]
To test with images with no labels, you may replace lines 165 - 167 with the following:

`annotation = numpy.zeros((image_np.shape[0], image_np.shape[1]))`

In this case, the prediction line and the predicted_softmax line would not need to be changed.
